### PR TITLE
Show Estimate Stats on Graphviz

### PR DESCRIPTION
**Pull request by fgwang7w**
Improve graphviz output with estimate stats.

Here is a sample output from tpch-q3
Before:
q3_before

After:
q3_after

```
== RELEASE NOTES ==

General Changes
* Improve graphviz output with estimate stats.
```

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java [hunk 1]
```diff
@@ -464,7 +472,7 @@ public Void visitLimit(LimitNode node, Void context)
         @Override
         public Void visitTableScan(TableScanNode node, Void context)
         {
-            printNode(node, format("TableScan[%s]", node.getTable()), NODE_COLORS.get(NodeType.TABLESCAN));
+            printNode(node, format("TableScan | [%s]", node.getTable()), NODE_COLORS.get(NodeType.TABLESCAN));
```
Comment (aaneja) [inline presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java:475]: Thanks for fixing this `TableScan` label formatting. Usually, `TableHandles` are long strings, so having this formatted with `|` will move these to the next line, improving readability

**Review by aaneja — APPROVED** [review]
LGTM

**Comment by fgwang7w** [comment]
@aaneja many thanks for giving 1st pass review. 
@yingsu00 could you pls help review and merge this fix for visual plan improvement? Thanks!

**Comment by NikhilCollooru** [comment]
Code changes look good. But both the `before` and `after` pictures has `Estimates : {...` ? :thinking: :thinking:

**Comment by fgwang7w** [comment]
> Code changes look good. But both the `before` and `after` pictures has `Estimates : {...` ? 🤔 🤔

Thanks for reviewing the code and PR description. I fixed the correct before fix png file, uploaded a different version before.

**Review by yingsu00 — APPROVED** [review]

**Pull request merged**